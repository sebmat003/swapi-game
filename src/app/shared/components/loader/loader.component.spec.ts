import { createComponentFactory, Spectator } from '@ngneat/spectator/jest';
import { LoaderComponent } from "./loader.component";

describe('LoaderComponent', () => {
  let spectator: Spectator<LoaderComponent>;
  const createComponent = createComponentFactory({
    component: LoaderComponent,
    detectChanges: false
  });

  beforeEach(() => spectator = createComponent())

  it('Should match snapshot', () => {
    spectator.detectChanges();

    expect(spectator.fixture).toMatchSnapshot();
  });
});
