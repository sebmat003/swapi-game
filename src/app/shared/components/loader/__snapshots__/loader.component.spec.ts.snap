// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoaderComponent Should match snapshot 1`] = `
<app-loader>
  <div
    class="text-center"
  >
    <div
      class="spinner-border text-primary"
      role="status"
    >
      <span
        class="visually-hidden"
      >
        Loading...
      </span>
    </div>
  </div>
</app-loader>
`;
